plugin/inlay_hint.py

```diff
@@ -0,0 +1,154 @@
+from .core.protocol import InlayHintLabelPart, MarkupContent, Point, InlayHint, Request
+from .core.registry import LspTextCommand
+from .core.sessions import Session
+from .core.typing import Optional, Union
+from .core.views import FORMAT_MARKUP_CONTENT, point_to_offset, minihtml
+from .formatting import apply_text_edits_to_view
+import html
+import sublime
+import uuid
+
+
+class LspInlayHintClickCommand(LspTextCommand):
+    capability = 'inlayHintProvider'
+
+    def run(self, _edit: sublime.Edit, session_name: str, inlay_hint: InlayHint, phantom_uuid: str,
+            event: Optional[dict] = None, label_part: Optional[InlayHintLabelPart] = None) -> None:
+        """
+        Insert textEdits for the given inlay hint.
+        If a InlayHintLabelPart was clicked, label_part will be passed as an argument to the LspInlayHintClickCommand
+        and InlayHintLabelPart.command will be executed.
+        """
```

Comment:
Just a nit but this is not really a public documentation but rather description of internal logic so IMO shouldn't be using a docblock syntax for it, just normal comments.